plugin/storage/kafka/factory_test.go

```diff
@@ -120,6 +120,7 @@ func TestKafkaFactoryDoesNotLogPassword(t *testing.T) {
 				"--kafka.producer.authentication=plaintext",
 				"--kafka.producer.plaintext.username=username",
 				"--kafka.producer.plaintext.password=SECRET",
+				"--kafka.producer.plaintext.mechanism=PLAINT",
```

Comment:
```suggestion
				"--kafka.producer.plaintext.mechanism=PLAIN",
```
why is this even needed? the mechanism defaults to PLAIN already

Reply:
Yes, actually, I forgot to remove it, this parameter is no point. I will remove it on the later commit.